### PR TITLE
EWL-6447 Themes 50/50 video hub card 

### PR DESCRIPTION
**Pull request by designerneil**
## JIRA Ticket(s)
- [EWL-6447: Bug: Hub page - 50/50 - theme](https://issues.ama-assn.org/browse/EWL-6447)

## Description:
Theme 50/50 video hub card

## To Test:
- switch your SG2 branch to `bugfix/EWL-6447-theme-50-50-video` (https://github.com/AmericanMedicalAssociation/ama-d8/pull/983)
- run `gulp serve`
- switch your D8 branch to `bugfix/EWL-6447-theme-50-50-video`
- enable local SG2 in your D8 instance
- run `drush @one.local cr`
- visit http://ama-one.local/node/24416
- edit the page
- add a 50/50 video hub card 
- save
- observe the 50/50 video hub card is now themed

## Automated Test:
n/a

## Style Guide:
The PR should look like this https://americanmedicalassociation.github.io/ama-style-guide-2/?p=molecules-hub-hero-as-fifty-fifty-video

## Relevant Screenshots/GIFs:
screen shot 2018-10-30 at 3 58 15 pm

## Additional Notes:
Anything more to add?

---

[Pull Request Guidelines](https://github.com/palantirnet/development_documentation/blob/master/guidelines/pr_review.md)

### CODE REVIEW

**Comment by damontgomery** [comment]
Questions left on the Drupal 8 pull request: https://github.com/AmericanMedicalAssociation/ama-d8/pull/983.

**Comment by mrspete** [comment]
removing priority label after conferring with @r-kitchen

File: styleguide/source/assets/scss/02-molecules/hub/_hub-card.scss [hunk 1]
```diff
@@ -33,13 +33,14 @@
 
   &__image {
     margin: 0;
+    height: 100%;
     display: flex;
     height: 100%;
```
Comment (damontgomery) [inline styleguide/source/assets/scss/02-molecules/hub/_hub-card.scss:38]: It looks like there are duplicate `height: 100%;` lines now.

**Review by damontgomery — COMMENTED** [review]
This looks pretty good.

Do we want the video to be slightly larger, or is this good for now?

larger_image

Tested with https://www.youtube.com/watch?v=K6PBY1S9SdE

I also added a comment about a duplicate line in CSS.

**Pull request merged**